priv/repo/.formatter.exs

```diff
@@ -1,4 +1,4 @@
 [
   import_deps: [:ecto_sql],
-  inputs: ["/**/*.{ex,exs}"],
```

Comment:
When I first ran `mix format` it just hung and I wasn't sure why. Once I learned about `.formatter.exs` and `subdirectories` (which the top-level file uses) I found this extra leading `/`. With that I'm pretty sure it was considering all files on my system for formatting.